.github/workflows/close-stale-issues.yml

```diff
@@ -12,25 +12,25 @@ env:
     and it is being closed. If you feel closing this issue is not the
     right thing to do, please leave a comment.
   WARN_MESSAGE: >
-    There has been no activity on this issue for
-    3 years and it may no longer be relevant.
-    It will be closed 1 month after the last non-automated comment.
+    There has been no activity on this issue for 90 days.
+    The help repository works best when sustained engagement moves conversation forward.
+    The issue will be closed 1 month after the last non-automated comment.
+    If you are still experiencing this issue on the latest supported versions of Node.js, please leave a comment.
 # yamllint enable
 
 jobs:
   stale:
     if: github.repository == 'nodejs/help'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@a20b814fb01b71def3bd6f56e7494d667ddf28da #4.1.1
```

Comment:
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions